### PR TITLE
[Quick Sign] Fix previous button when navigating between pages

### PR DESCRIPTION
**Pull request by bnchi**
When a user is at the end of the pdf pages the previous button is set to disabled and the button doesn't work anymore this should be a small fix.

### CODE REVIEW

**Review by mkhatib — APPROVED** [review]

**Comment by mkhatib** [comment]
@bnchi let me know if you can't merge the PR after I approve it

**Comment by bnchi** [comment]
@mkhatib I'm not able to merge, here's my email in case you need to add bnchifahmi@gmail.com

**Comment by mkhatib** [comment]
Just added you to the repo collaborators, can you merge now?

**Pull request merged**
